api/python/src/cellxgene_ontology_guide/supported_versions.py

```diff
@@ -61,8 +68,10 @@ def __init__(self, version: Optional[str] = None):
         ontology_info = load_supported_versions()
         if version is None:
             version = get_latest_schema_version(ontology_info.keys())
-        elif version not in ontology_info:
-            raise ValueError(f"Schema version {version} is not supported in this package version.")
+        else:
+            version = str(coerce_version(version))
```

Comment:
nit: very small, but I think it's better practice to assign self.version = version up top and do this (potential) transformation on self.version rather than the input str

Reply:
I understand that it is not best practice to manipulate the input variables. I'll instead save any manipulation to a different variable `_version`. I don't want to assign to `self.version` right off the bat, because it will mean using more `self.version` lookups later in the function and i'd like to avoid doing more `.` lookups than needed. Do you think it detract from readability the way it is currently written?